style.css

```diff
@@ -0,0 +1,137 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
- [ ] Also make sure you use the correct fonts as requested by the previous reviewer. `Poppins`

Hint: Since it's not a default font for browsers, You may consider importing it on `google Fonts`. you can find [here](https://www.youtube.com/results?search_query=how+to+use+google+fonts+in+css) how to use google fonts